.dagger/dagger.cue

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
+	"github.com/prometheus/promci"
```

Comment:
I would apply the same logic than in go to sort the import here.

all `dagger` import should come first. Then you have an empty line, and then you have the rest.